api/api.js

```diff
@@ -37,7 +37,7 @@ schedule.scheduleJob('0 3 * * *', function(){
     var gzip = zlib.createGzip();
     var today = new Date();
     var wstream = fs.createWriteStream('/pheromon/data/backups/' + today.getDay() + '.txt.gz');
-    var proc = spawn('pg_dump', ['-p', process.env.DB_PORT_5432_TCP_PORT, '-h', process.env.DB_PORT_5432_TCP_ADDR, '-U', process.env.POSTGRES_USER, '-d', process.env.POSTGRES_USER, '-w']);
+    var proc = spawn('pg_dump', ['-p', process.env.POSTGRES_PORT, '-h', process.env.POSTGRES_HOST, '-U', process.env.POSTGRES_USER, '-d', process.env.POSTGRES_USER, '-w']);
```

Comment:
Finally, the `--net=host` can be avoided by using the docker host address (`172.X.0.1`).

We just need to find a way to get this address programmatically.